agent/session.go

```diff
@@ -31,26 +31,26 @@ type session struct {
 	conn *grpc.ClientConn
 	addr string
 
-	agent     *Agent
-	sessionID string
-	session   api.Dispatcher_SessionClient
-	errs      chan error
-	messages  chan *api.SessionMessage
-	tasks     chan *api.TasksMessage
+	agent       *Agent
+	sessionID   string
+	session     api.Dispatcher_SessionClient
+	errs        chan error
+	messages    chan *api.SessionMessage
+	assignments chan *api.AssignmentsMessage
```

Comment:
I guess when we add backward compatibility, one way to do it would be to create a fake `COMPLETE` `Assignments` message that has uses a task list from the `Tasks` RPC. Or we could have a separate channel for tasks. But I think a single channel might be cleaner.